t3/common.py

```diff
@@ -17,6 +17,7 @@
 DATA_BASE_PATH = os.path.join(t3_path, 'tests', 'data')
 EXAMPLES_BASE_PATH = os.path.join(t3_path, 'examples')
 PROJECTS_BASE_PATH = os.path.join(t3_path, 'Projects')
+test_minimal_project_directory = os.path.join(PROJECTS_BASE_PATH, 'test_minimal_delete_after_usage')
```

Comment:
Yes we use it in `test_main.py`, but since we only use it in `test_main.py`, I will define the path there instead.

The commit previously had moved that function, but one of your [PRs](https://github.com/ReactionMechanismGenerator/T3/commit/7a32790423e88e9dfd88822517f3c0bde11514a2) ended up doing that too so when I had rebased this branch, I had forgotten to update the commit message. I'll remove that phrase.